Example-CustomUserGroups-REXX/USRGRP01.rexx

```diff
@@ -0,0 +1,2980 @@
+     * loop in the event HWIHOST(OFF) fails and invokes FatalError,
+     * which will goes through this path again
+     */
+    HWIHOST_ON = FALSE
+    hwiHostRC = hwihost("OFF")
+    Say 'HWIHOST("OFF") return code is: ('||hwiHostRC||')'
+
+    If hwiHostRC <> 0 Then
+      Exit FatalErrorAndCleanup('** unable to turn off HWIHOST **')
+  End
+
+Return 0
+
+/**********************************************************************
+ * Function: SurfaceResponse
+ *
+ * Purpose: Parses through the response parm and if the request failed
+ *          showcase the issue
+ *
+ * Input: None
+ *
+ * Output: None
+ *
+ * Side-Effects: Displays the response data
+ *
+**********************************************************************/
+SurfaceResponse:
+
+  Say
+  Say 'Rexx RC: (' || Arg(1) || ')'
+
+  /*
+   * Continue processing even if RC <> 0 because additional information
+   * could have been returned in the response to help understand the
+   * error
+   */
+
+  Say 'HTTP Status: (' || response.httpstatus || ')'
+  successIndex = Index(response.httpstatus, '2')
+
+  /* SE responded successfully */
+  If successIndex = 1 Then Do
+    Say 'SE DateTime: (' || response.responsedate || ')'
+    Say 'SE requestId: (' || response.requestId || ')'
+
+    If response.httpstatusNum = '201' Then
+      Say 'Location Response: (' || response.location || ')'
+
+    If  response.responsebody <> '' Then Do
+      Say 'Response Body: (' || response.responsebody || ')'
+      Return response.responsebody
+    End
+  End
+  Else Do
+    Say 'Reason Code: (' || response.reasoncode || ')'
+
+    If response.responsedate <> '' Then
+      Say 'SE DateTime: ('||response.responsedate||')'
+
+    If response.requestId <> '' Then
+      Say 'SE requestId: (' || response.requestId || ')'
+
+    If response.responsebody <> '' Then Do
+      Call ParseJSONData response.responsebody
+
+      If RESULT <> 0 Then Do
+        Say 'failed to parse response'
+      End
+      Else Do
+
+        bcpiiErr=FindJSONValue(0, "bcpii-error", HWTJ_BOOLEAN_TYPE)
+
+        errMsg = '*** SE generated error message:'
+
+        If bcpiiErr = 'true' Then
+          errMsg = '*** BCPii generated error message:'
+
+        Say errMsg
+
+        errmessage = FindJSONValue(0,"message", HWTJ_STRING_TYPE)
+        Say '(' || errmessage || ')'
+        Say
+
+        Say 'Complete Response Body: (' || response.responsebody || ')'
+      End
+    End
+  End
+
+Return ''
+
+/**********************************************************************
+ * Function: Usage
+ *
+ * Purpose: Provide Usage guidance to the invoker
+ *
+ * Input: whyString - A string describing why the user is receiving this
+ *                    usage message
+ *
+ * Output: -1 to indicate fatal script error.
+ *
+ * Side-Effects:
+ *
+ *  1. Displays an informational message describing how one would use
+ *     the script
+ *
+**********************************************************************/
+Usage:

```

Comment:
I would add a command line option to call this usage function to display the help text with a -h option like most standard cli